<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_SendInput">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_InputType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mouse Input</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Keybord Input</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hardware Input</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Mouse</string>
    </attribute>
    <widget class="QSpinBox" name="spinBox_MouseX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MouseY">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>flag</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Absolute">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_ABSOLUTE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_HWheel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_HWHEEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>data</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Data">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10000</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Move">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_MOVE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_MoveNoCoalesce">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_MOVE_NOCOALESCE</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LeftDown">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_LEFTDOWN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LeftUp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_LEFTUP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_RightDown">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_RIGHTDOWN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_RightUp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_RIGHTUP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_MiddleDown">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_MIDDLEDOWN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_MiddleUp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_MIDDLEUP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_VirtualDesk">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_VIRTUALDESK</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Wheel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_WHEEL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_XDown">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_XDOWN</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_XUp">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSEEVENTF_XUP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>é¡µ</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay(s)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Delay">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
